test/bin/build_images.sh

```diff
@@ -376,12 +377,47 @@ do_group() {
         done
     fi
 
-    if (( ${#buildid_list[@]} )); then
+    if ${BUILD_INSTALLER} && ! ${COMPOSER_DRY_RUN}; then
+        for download_file in "${groupdir}"/*.image-fetch; do
```

Comment:
Since there is no build step for these, would it make sense to put them in their own group or process them separately instead of mingling them in with the groups of image build instructions?

Reply:
The pro for putting them together with everything else is that images are fetched in parallel together with osbuilder running. It can take up to 5-10m to download an .iso, which I thought would be a waste.